sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LegacyServiceClient.cs

```diff
@@ -16,87 +19,1378 @@ namespace Azure.AI.TextAnalytics.ServiceClients
     ///
     internal class LegacyServiceClient : ServiceClient
```

Comment:
Essentially, this is the `TextAnalyticsClient` from the previous version.   Some types have been remapped as the legacy generated surface is entirely internalized.